Sources/BraveShields/ShieldStrings.swift

```diff
@@ -146,3 +146,35 @@ public extension Strings.Shields {
     comment: "A button that ignores the brave player"
   )
 }
+
+// MARK: - Blocked Page
+
+public extension Strings.Shields {
+  /// A tab title that appears when a page was blocked
+  static let domianBlockedTitle = NSLocalizedString(
+    "DomainBlockedTitle", tableName: "BraveShared", bundle: .module,
+    value: "Domain Blocked",
+    comment: "A tab title for the warning page that appears when a page was blocked"
+  )
+  
+  /// A title in the warning page that appears when a page was blocked
+  static let domianBlockedPageTitle = NSLocalizedString(
+    "DomianBlockedPageTitle", tableName: "BraveShared", bundle: .module,
+    value: "This Site May Attempt to Track You Across Other Sites",
+    comment: "A title in the warning page that appears when a page was blocked"
+  )
+  
+  /// A title in the warning page that appears when a page was blocked
+  static let domianBlockedPageMessage = NSLocalizedString(
+    "DomianBlockedPageMessage", tableName: "BraveShared", bundle: .module,
+    value: "Brave has prevented the following site from loading:",
+    comment: "A message in the warning page that appears when a page was blocked"
+  )
+  
+  /// A description in the warning page that appears when a page was blocked
+  static let domianBlockedPageDescription = NSLocalizedString(
+    "DomianBlockedPageDescription", tableName: "BraveShared", bundle: .module,
```

Comment:
`Domian*` -> `Domain` typo in 3 strings above

Reply:
Oopsies, thanks for the catch

Reply:
Fixed